inc/field.class.php

```diff
@@ -348,10 +341,10 @@ function showSummary($container) {
 
                echo '<td class="rowhandler control center">';
                echo '<div class="drag row" style="cursor:move;border:none !important;">';
-               echo '<img src="../pics/drag.png" alt="#" title="' . __('Move') .'" width="16" height="16" />';
+               echo '<img src="../pics/drag.png" alt="#" title="'.__('Move').'" width="16" height="16">';
```

Comment:
Standalone markup is generally closed ;)

Reply:
in html5, it's optional

Reply:
Yes it is... But it is also considered as a good practice to close all tags. Well this is not important.